src/build.d

```diff
@@ -163,30 +163,37 @@ auto lexer()
 /// Returns: the dependency that generates the dmd.conf file in the output folder
 auto dmdConf()
 {
-    // TODO: add support for Windows
-    string exportDynamic;
-    version(OSX) {} else
-        exportDynamic = " -L--export-dynamic";
-
-    auto conf = `[Environment32]
-DFLAGS=-I%@P%/../../../../../druntime/import -I%@P%/../../../../../phobos -L-L%@P%/../../../../../phobos/generated/{OS}/{BUILD}/32{exportDynamic}
-
-[Environment64]
-DFLAGS=-I%@P%/../../../../../druntime/import -I%@P%/../../../../../phobos -L-L%@P%/../../../../../phobos/generated/{OS}/{BUILD}/64{exportDynamic} -fPIC`
-        .replace("{exportDynamic}", exportDynamic)
-        .replace("{BUILD}", env["BUILD"])
-        .replace("{OS}", env["OS"]);
+    const makedmdconfBin = env["G"].buildPath("makedmdconf").exeName;
+    Dependency buildMakedmdconf = {
+        name: "(DC) MAKEDMDCONF",
+        target: makedmdconfBin,
+        sources: [srcDir.dirName.buildPath("makedmdconf.d")],
+        command: [
+            env["HOST_DMD_RUN"],
+            "-of$@",
+            "$<"
+        ],
+    };
+    buildMakedmdconf.run;
 
-    auto target = env["G"].buildPath("dmd.conf");
-    auto commandFunction = (){
-        conf.toFile(target);
-    }; // defined separately to support older D compilers
-    Dependency dependency = {
-        target: target,
-        name: "(TX) DMD_CONF",
-        commandFunction: commandFunction,
+    string[] options = new string[0];
```

Comment:
```suggestion
    string[] options;
```

There's no difference and the appends below will trigger a resize allocation anyhow.